src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+    return text
+
+
+def swallow_angle_brackets(text: str) -> str:
+    """
+    Disappear text in and surrounding an angle bracket
+    >>> text = " <O> mea dext<e>ra illa CICERO RUFO Quo<quo>. modo proficiscendum <in> tuis.  deesse HS <c> quae    metu <exagitatus>, furore   <es>set consilium  "
+    >>> swallow_angle_brackets(text)
+    'mea  illa CICERO RUFO  modo proficiscendum  tuis.  deesse HS  quae    metu  furore    consilium'
+
+    """
+    text = swallow(text, ANGLE_BRACKETS)
+    # There are occasionally some unmatched ANGLE_BRACKETS
+    text = text.replace("<", " ")
+    text = text.replace(">", " ")
+    return text
+
+
+def disappear_angle_brackets(text: str) -> str:
+    """
+    Remove all angle brackets, keeping the surrounding text; no spaces are inserted
+    :param text: text with angle bracket
+    :return: text without angle brackets
+    """
+    text = text.replace("<", "")
+    text = text.replace(">", "")
+    return text
+
+
+def swallow_square_brackets(text: str) -> str:
+    """
+    Swallow text inside angle brackets, without substituting a space.
+    :param text: text to clean
+    :return: text without square brackets and text inside removed
+
+    >>> swallow_square_brackets("qui aliquod institui[t] exemplum")
+    'qui aliquod institui exemplum'
+
+    >>> swallow_square_brackets("posthac tamen cum haec [tamen] quaeremus,")
+    'posthac tamen cum haec  quaeremus,'
+
+    """
+    return swallow(text, SQUARE_BRACKETS)
+
+
+def swallow_obelized_words(text: str) -> str:
+    """
+    Swallow obelized words; handles enclosed and words flagged on the left.
+    Considers plus signs and daggers as obelization markers
+    :param text: Text with obelized words
+    :return: clean text
+
+    >>> swallow_obelized_words("tu Fauonium †asinium† dicas")
+    'tu Fauonium  dicas'
+
+    >>> swallow_obelized_words("tu Fauonium †asinium dicas")
+    'tu Fauonium dicas'
+
+    >>> swallow_obelized_words("meam +similitudinem+")
+    'meam'
+
+    >>> swallow_obelized_words("mea +ratio non habet" )
+    'mea non habet'
+
+    """
+    text = swallow(text, OBELIZED_WORDS)
+    text = swallow(text, OBELIZED_WORD)
+    text = swallow(text, OBELIZED_PLUS_WORDS)
+    return swallow(text, OBELIZED_PLUS_WORD)
+
+
+def disappear_round_brackets(text: str) -> str:
+    """
+    Remove round brackets and keep the text intact
+    :param text: Text with round brackets.
+    :return: Clean text.
+
+    >>> disappear_round_brackets("trib(unus) mil(itum) leg(ionis) III")
+    'tribunus militum legionis III'
+    """
+    text = text.replace("(", "")
+    return text.replace(")", "")
+
+
+def swallow_editorial(text: str) -> str:
+    """
+    Swallow common editorial morks
+    :param text: Text with editorial marks
+    :return: Clean text.
+
+    >>> swallow_editorial("{PRO P. QVINCTIO ORATIO} Quae res in civitate trib(unus) mil(itum) leg(ionis) III tu Fauonium †asinium† dicas meam +similitudinem+  mea +ratio non habet ...     ")
```

Comment:
This is a case where I am pretty sure round and curly brackets are used by PHI for markup. Not a deal breaker, and the way you have modularized the code makes this easily customizable in the future!